homeassistant/components/smart_meter_b_route/coordinator.py

```diff
@@ -0,0 +1,61 @@
+"""DataUpdateCoordinator for the Smart Meter B-route integration."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from momonga import Momonga, MomongaError
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    ATTR_API_INSTANTANEOUS_CURRENT_R_PHASE,
+    ATTR_API_INSTANTANEOUS_CURRENT_T_PHASE,
+    ATTR_API_INSTANTANEOUS_POWER,
+    ATTR_API_TOTAL_CONSUMPTION,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BRouteUpdateCoordinator(DataUpdateCoordinator[dict[str, Any]]):
+    """The B Route update coordinator."""
+
+    api: Momonga
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        device: str,
+        id: str,
+        password: str,
+        update_interval: timedelta,
```

Comment:
This is static, why do we include it in the parameters?

Reply:
I just followed the way found in other integrations, I can change it